spec/AutomatedLinkAcceptanceProxy.yaml

```diff
@@ -1711,13 +1689,12 @@ paths:
                     ClientContainerUuid = ClientStructure.ClientLtp
```

Comment:
The name of this callback could be updated from LtpName to a common term , since now it has become common to both air-interface (external-label) and wire-interface(original-ltp-name).

Reply:
Callback name changed from LtpName to LtpDesignation